File: javaClient/src/main/java/cb/bots/UserBot.java
package cb.bots;

public class UserBot implements BotAi {
    private String name;
    private int id;

    public UserBot() {
        // Wähle hier deinen Nickname aus (ändere "UserAI")
        name = "UserAI";
        // Die Token ID wird vom Server vergeben.. nicht editieren!
        id = -1;
    }

    @Override
    public int play(int[][] field) {
        /*
        Implementiere hier deine Logik, damit die KI spielen kann
         */
        return 0; // aktuell wird immer die erste Spalte ausgewählt als nächsten Zug
    }

    @Override
    public String getName() {
        return name;
    }

    @Override
    public void setPlayerId(int playerId) {
        id = playerId;
    }

    @Override
    public int getPlayerId() {
        return id;
    }
}

File: csharpClient/Bots/UserBot.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Linq;

namespace CsClient.Bots
{
    public class UserBot : IBot
    {
        /// <inheritdoc/>
        public string Name { get; }

        /// <inheritdoc/>
        public int PlayerId { get; set; }

        /// <inheritdoc/>
        public int Play(int[][] field)
        {
            //
            // Deine Play Methode - hier kann deine Logik stehen :)
            // 

            return 0;
        }

        public UserBot()
        {
            Name = "FOOBAR";      // Change
            PlayerId = -1;        // DO not Change
        }
    }
}
